droneapi/module/api.py

```diff
@@ -450,6 +455,7 @@ def mavlink_packet(self, m):
             self.level = m.battery_remaining
             self.__on_change('battery')
         elif typ == "HEARTBEAT":
+            self.system_status = m.system_status
             self.__on_change('mode', 'armed')
```

Comment:
I think its only use is to have the ability to add a attribute_observer. I think you should add it to keep consistent will the other attributes.